crates/axerrno/src/lib.rs

```diff
@@ -26,6 +26,8 @@ pub use linux_errno::LinuxError;
 #[non_exhaustive]
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum AxError {
+    /// The address is used
```

Comment:
More detail comments from rust std: `A socket address could not be bound because the address is already in use elsewhere.`